src/main/resources/templates/leaderboard.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org" th:include = "layout :: page">
```

Comment:
Needs to be placed in sub directory, and if this is just the default view. Will probably be called index.html